content/en/blog/_posts/2024-secret-sync-controller-alpha.md

```diff
@@ -0,0 +1,56 @@
+
+5. Apply the CRDs in the cluster and secrets will automatically begin syncing.  
+
+## Security features 
+By default, the Secret Sync Controller (SSC) enables the following security features: 
+
+1. Isolation of Functionality: The SSC splits out the functionality of syncing secrets from a cloud provider (via a secret sync Provider) and storing the secrets in a cluster (the SSC). Each component can have much more limited permissions, minimizing the risk of a cluster compromise.  
+
+2. Limited Permissions via ValidatingAdmissionPolicies: The SSC is a privileged facility that should be in a protected namespace, such as `kube-system`. The SSC leverages ValidatingAdmissionPolicies to limit its capabilities, ensuring it cannot perform operations such as deleting Secrets, or editing Secrets beyond its designated scope (defined via labels and annotations). Validating admission policies will be generally available and enabled by default starting with Kubernetes 1.30; right now, they are beta and opt-in.
+
+3. Cloud Provider vs Workload Configuration: Configuration of the secrets to be synced to the cluster are split into two layers – the cloud provider configuration and the workload specific configuration. This allows a cloud provider admin to create and define the cloud provider secret storage and cloud identity that has access to secrets. The workload developer can then define how the secret is stored in the cluster (name, namespace, etc) and which local Kubernetes identity has access to the secret.  
+
+4. Namespace Boundaries: The SSC leverages the Kubernetes namespace security boundary to isolate Secrets so that only Pods in a namespace have access to Secrets in that namespace. 
+
+5. Federated Identity for Syncing Secrets: The SSC does not have its own global identity for accessing secrets. Instead, at deployment time, the admin configures a unique local identity (ServiceAccount) for each workload that uses secrets, and can federate this with a unique cloud identity that has access to the Secrets it requires.
+
+In addition to the previous on by default security features, a cluster admin should enable the following security features: 
+
+6. Owner Reference Permission Enforcement: The SSC permissions can be further limited by enabling the _Owner Reference Permission Enforcement_ admission controller, which ensures that only the owners of a Secret have permissions to update and delete the Secret.
+
+7. Encryption: Kubernetes secrets are stored encoded in etcd, and the secret storage should be encrypted. For example, a cluster admin should enable a KMS plugin for the cluster to encrypt secret data.  
+
+## What’s next? 
+Depending on feedback and adoption, the SIG Auth and Secret Store CSI maintainers plan to push the Secret Sync Controller to beta later in 2024.   
```

Comment:
Let's add some more detail about (1) merging the PR after review (2) Moving all the code for sync to a new repo (3) graduating to beta based on feedback.